examples/youtube_tracker/youtube_tracking_example.html

```diff
@@ -1,3 +1,11 @@
+<!--
+This is a example webpage that includes our youtube tracking script. It will track the linked youtube video and send events realted to it to our snowplow mini collector
```

Comment:
```suggestion
This is an example webpage that includes our YouTube tracking script. It will track the linked YouTube video and send events related to it to our snowplow mini collector.
```